src/app/clients/clients-view/general-tab/general-tab.component.html

```diff
@@ -54,10 +54,10 @@ <h3>Upcoming Charges</h3>
     <ng-container matColumnDef="Actions">
       <th mat-header-cell *matHeaderCellDef> Actions </th>
       <td mat-cell *matCellDef="let element">
-        <button class="account-action-button" mat-raised-button color="primary">
+        <button class="account-action-button" mat-raised-button color="primary" [routerLink]="['../','paycharge',element.id]">
```

Comment:
You should implement pay charge form as a part of this issue.
![Screenshot from 2020-05-10 23-49-29](https://user-images.githubusercontent.com/54709463/81507146-f230bd00-9318-11ea-8398-5976bab84eb1.png)